docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc

```diff
@@ -7,38 +7,70 @@
 
 The `inputs` section of the `elastic-agent.yml` file specifies how {agent} locates and processes input data.
 
-By default {agent} collects system metrics, such as CPU, memory, network, and file system metrics, and sends them to the default output. For example, to define the datastream for `cpu` metrics, this is the configuration:
+* <<elastic-agent-input-configuration-sample-metrics>>
+* <<elastic-agent-input-configuration-sample-logs>>
+
+[discrete]
+[[elastic-agent-input-configuration-sample-metrics]]
+== Sample metrics input configuration
+
+By default {agent} collects system metrics, such as CPU, memory, network, and file system metrics, and sends them to the default output. For example, to define datastreams for `cpu`, `memory`, `network` and `filesystem` metrics, this is the configuration:
 
 ["source","yaml"]
 -----------------------------------------------------------------------
-- id: unique-system-metrics-id <1>
-  type: system/metrics <2>
-  use_output: default <3>
-  meta:
-    package: <4>
-      name: system
-      version: 0.10.9
-  data_stream:
-    namespace: default <5>
+- type: system/metrics <1>
+  id: unique-system-metrics-id <2>
+  data_stream.namespace: default <3>
```

Comment:
The flattened notation has not been released yet. We ended up reverting the Beats PR https://github.com/elastic/beats/pull/36610 due to a bug and now we're re-implementing it in the Elastic-Agent itself: https://github.com/elastic/elastic-agent/pull/3465.

We just need to make sure this version of the docs are released together with the feature.

Reply:
I added a note for myself in the description to wait until the Agent PR is merged.